admin-notice/js/script.js

```diff
@@ -0,0 +1,44 @@
+function tryGetNoticeContainer(currentElement) {
+    if (currentElement === null || currentElement.hasAttribute(dismissal_data.data_attribute)) {
+        return currentElement;
+    }
+    return tryGetNoticeContainer(currentElement.parentElement);
+}
+
+function persistDismissedNotice(containerElement) {
+    const dismissIdentifier = containerElement?.getAttribute(dismissal_data.data_attribute);
```

Comment:
@pschoffer @GaryJones  This has introduced regressions into VIP sites. The file isn't being transpiled. It's breaking the admin for some users on older chrome.

https://github.com/Automattic/vip-go-mu-plugins-built/blob/master/admin-notice/js/script.js#L9